src/app/controllers/OverviewController.ts

```diff
@@ -148,11 +149,22 @@ export default class OverviewController {
 		}
 
 		let attrEffectiveness = 0
+		let displayAttrEffectiveness = false
 		if (activeValidators.length > 0) {
-			attrEffectiveness = sumBigInt(validators, (cur) => (cur.attrEffectiveness ? new BigNumber(cur.attrEffectiveness.toString()) : new BigNumber(0)))
+			displayAttrEffectiveness = true
+
+			attrEffectiveness = sumBigInt(activeValidators, (cur) =>
```

Comment:
FYI: This change (`validators` => `activeValidators`) fixes the calculation.

Reply:
I need everything I came for 
#datakeova

Reply:
#datakeova

Reply:
I’m a active validator 
#datakeova

Reply:
I am a active validator